lib/uvobot/notifications/discourse_notifier.rb

```diff
@@ -46,7 +46,7 @@ def detail_message(detail)
         if detail
           "**Cena:** #{detail[:amount]} EUR  "
         else
-          "**Detaily sa nepodarilo extrahovať.**  "
+          '**Detaily sa nepodarilo extrahovať.**  '
```

Comment:
Markdown linebreak. "When you do want to insert a <br /> break tag using Markdown, you end a line with two or more spaces, then type return."